src/components/mol.list-users/list-user.component.tsx

```diff
@@ -34,14 +38,15 @@ export const ListUsers = (): JSX.Element => {
   return (
     <section>
       <h1>{listUserStrings.title}</h1>
-
-      <Button text="Novo Usuário" onClick={Createuser} />
-
+      <div>
+        <Button text="Novo Usuário" onClick={Createuser} />
```

Comment:
```suggestion
        <Button text="Novo usuário" onClick={Createuser} />
```

por que usar maiúscula?

Reply:
foi ajustado